website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+// All of the following configs are used in the website and documentation provided by the Design team Glyph
+
+// Colors
+
+@mixin detox-colors($prefix) {
```

Comment:
Using mixins is not correct AT ALL here. You want to use mixins when there is some repetitive aspect. 

Why would you want to use those color mixins more than twice? You define them once in `:root` as CSS variables and there you go. Every descendant can access them via `var(...)`.